lib/runner.js

```diff
@@ -46,6 +50,12 @@ util.inherits(Runner, EventEmitter);
 module.exports = Runner;
 
 optionChain(chainableMethods, function (opts, title, fn) {
+	if (opts.todo && typeof title === 'function') {
+		throw new TypeError('Expected a string');
+	} else if (opts.todo) {
+		fn = noop;
+	}
```

Comment:
I'd prefer a single `if (opts.todo) {}`, then nest the `title` check. Took me a while to figure out `fn = noop` was used when `title` was not a function :smile: